openrpc/src/stellar-rpc/contentDescriptors/Format.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Format": {
+    "name": "format",
+    "summary": "indicates which the response format to return - json or xdr",
+    "description": "Lets the user choose the format in which the response should be returned - either as json or base64 encoded xdr strings.",
```

Comment:
```suggestion
    "description": "Lets the user choose the format in which the response should be returned - either as unpacked JSON or as base64-encoded XDR strings. Note that you should not rely on any schema for the JSON, as it will change when the underlying XDR changes.",
```